cumulus/parachains/runtimes/people/people-rococo/src/people.rs

```diff
@@ -0,0 +1,225 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use super::*;
+use codec::{Decode, Encode, MaxEncodedLen};
+use enumflags2::{bitflags, BitFlags};
+use frame_support::{
+	parameter_types, traits::ConstU32, CloneNoBound, EqNoBound, PartialEqNoBound,
+	RuntimeDebugNoBound,
+};
+use pallet_identity::{Data, IdentityInformationProvider};
+use parachains_common::impls::ToParentTreasury;
+use scale_info::{build::Variants, Path, Type, TypeInfo};
+use sp_runtime::{traits::AccountIdConversion, RuntimeDebug};
+use sp_std::prelude::*;
+
+parameter_types! {
+	//   27 | Min encoded size of `Registration`
+	// - 10 | Min encoded size of `IdentityInfo`
+	// -----|
+	//   17 | Min size without `IdentityInfo` (accounted for in byte deposit)
+	pub const BasicDeposit: Balance = deposit(1, 17);
+	pub const ByteDeposit: Balance = deposit(0, 1);
+	pub const SubAccountDeposit: Balance = deposit(1, 53);
```

Comment:
When adding a sub identity, the `BoundedVec` in the value of `SubsOf` grows by one `AccountId` and we add another `AccountId`, `(AccountId, Data)` key-value pair into `SuperOf`. `SubsOf` grows linearly with bytes, `SuperOf` adds entries to the map (we can assume 32 bytes for `Data`, worst case). It's probably out of scope for this PR, but it may be worth it to implement the same "base + byte deposit" logic here, WDYT?

Reply:
Yeah I was considering this too. Getting the `items` right is most important, which this does, but adding the bytes amount would make it consistent.